src/dbcp/data_mart/projects.py

```diff
@@ -274,7 +275,7 @@ def _get_proprietary_proposed_offshore(engine: sa.engine.Engine) -> pd.DataFrame
     )
     -- join the project, state, and county stuff
     SELECT
-        assoc.project_id,
+        proj.project_id,
```

Comment:
Projects that don't have a location were being dropped because of the `INNER JOIN` and pulling the `project_id`s from the association table.